R/makePolarPlot.R

```diff
@@ -0,0 +1,41 @@
+#' @keywords internal
+#' 
+#' @importFrom dplyr group_by summarize arrange desc pull mutate
+#' @importFrom rlang sym :=
+#' @importFrom ggplot2 ggplot aes geom_col coord_polar facet_wrap 
+#'   theme_minimal theme element_blank
+#' 
+.makePolarPlot <- function(df, idCol, metricCol, valueCol, weightCol, 
```

Comment:
Would it make sense to have this and the other functions `@export`ed? Not that I am expecting too many persons to use them as they are, but maybe it does look better than I think?